libobs/obs-nix.c

```diff
@@ -707,6 +714,56 @@ static inline bool fill_keycodes(struct obs_core_hotkeys *hotkeys)
 	return error != NULL || reply == NULL;
 }
 
+static xcb_screen_t *default_screen(obs_hotkeys_platform_t *context,
+		xcb_connection_t *connection)
+{
+	int def_screen_idx = XDefaultScreen(context->display);
+	xcb_screen_iterator_t iter;
+
+	iter = xcb_setup_roots_iterator(xcb_get_setup(connection));
+	while (iter.rem) {
+		if (def_screen_idx-- == 0) {
+			return iter.data;
+			break;
```

Comment:
Isn't this `break` unreachable?

Reply:
Urgh.. Yeah. Unecessary. Will update in a moment

Reply:
In that case, I think the `if` statement does not need curly braces if it only contains a single-line statement.